python/_eos.cc

```diff
@@ -298,6 +299,30 @@ BOOST_PYTHON_MODULE(_eos)
         .def("__str__", &Options::as_string)
         ;
 
+    // Units
+    class_<Unit, boost::noncopyable>("Unit", R"(
+            Represents the unit of the observables.
+
+            Seven possible entries are currently implemented in EOS:
+            Undefined, None, GeV, GeV2, InverseGeV2, InverseGeV4, InversePicoSecond.
```

Comment:
Can you make this comment a list? It will be rendered as such in the documentation. You can check the documentation locally with
```
make install
make -C doc html
```
The doc will be available as ``doc/_build/html/index.html``.